src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -135,7 +138,7 @@ slots:
     examples:
       - value: "Alzheimer"
         description: A string referring to some thing.
-  literal_datatype:
+  subject_literal_datatype:
```

Comment:
The only new field, the rest can be reused.

Reply:
Is that really necessary? Do we expect people to map other values than pure strings (like, integers or dates)?

Reply:
This specific one may not be, but other like `subject_literal_language` may become relevant quickly. But I can be convinced to keep things essential and drop it entirely. I kept it mostly to illustrate that I want to be able to extend the slots for the literal without touching the rest of the standard..

Reply:
Just noting that if we decided to reuse `subject_label` rather than creating `subject_literal`, we could add a `subject_label_language` which could _also_ be meaningful for non-literal mappings…